### ISSUE TITLE
xcodebuild tries to load the plugin, print error on fail

### ISSUE THREAD

**Issue by Cubox-** [insanehunter/XCode4_beginning_of_line#30]
Hi!

When using xcodebuild, I get a message each time telling me that Xcode_beginning_of_line failed to launch. While normal for a command line build, the extra output line can be confusing.

I am using Xcode 6.1.1

Thanks!

**Comment by insanehunter** [comment]
Agree, I think we can safely get rid of this message. Maybe wrap the line with `NSLog` in `#if DEBUG` macro?
